pkgs/development/libraries/onnxruntime/default.nix

```diff
@@ -62,20 +123,27 @@ stdenv.mkDerivation rec {
   ]);
 
   buildInputs = [
+    Foundation
```

Comment:
This needs to be put behind an `stdenv.isDarwin` conditional, otherwise we get evaluation errors on linux.